new/PRS13943BakweraSeyon.xml

```diff
@@ -51,7 +51,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <nationality type="Ethiopia"/>
                     <birth/>
                     <death/>
-                    <floruit>mid-14th mid 15th century</floruit>
+                    <floruit notBefore="1350" notAfter="14549">mid-14th mid 15th century</floruit>
```

Comment:
1449 or 1450

Reply:
Yes, I was in doubt how to write. In the @thea-m suggestion, there is 1449.
Does the first hlf of the 15th century  corresponds to 1449 or 1450?

Reply:
* correspond

Reply:
these numbers are not visible, they are only useful for filtering down in searches
1350-1449 or 1350-1450 are both ok but 1449 is more precise if we say that 1450 is the beginning of the second half

Reply:
Ok, thanks.